packages/ui-library/src/components/token-picker-button/index.tsx

```diff
@@ -17,11 +17,17 @@ const StyledTokenPickerButton = styled(Button).attrs({ variant: 'outlined' })`
   border-radius: ${spacing(15)};
   transition: box-shadow 300ms;
   background-color: ${colors[palette.mode].background.secondary};
-  border: 1.5px solid ${colors[palette.mode].border.border1};
+  border: 1px solid ${colors[palette.mode].border.border1};
   box-shadow: ${colors[palette.mode].dropShadow.dropShadow100};
   :disabled {
     box-shadow: none;
   }
+
+  &:hover {
+    background-color: ${colors[palette.mode].background.tertiary};
+    border: 1px solid ${colors[palette.mode].border.border1};
+    box-shadow: ${colors[palette.mode].dropShadow.dropShadow200};
```

Comment:
Should we? there are a lot of places were we want to use 100 and others where we want to use 200

Will take the transition comment tho :)

Reply:
I agree with you, I meant to edit colors.ts 😄. Currently all our dropshados are 100 because of this:
![Screenshot 2024-05-23 at 7 15 29 PM](https://github.com/Balmy-protocol/dca-fe/assets/144699868/34ec5589-b1d6-405e-8500-376a1848806f)